src/navigation/navigationhandler_lua.inl

```diff
@@ -254,43 +285,96 @@ namespace {
     );
 }
 
+// @TODO: Should this be a concrete Type in the documentation? Could also be used in bind
```

Comment:
Should we do this?  Would require rewriting the [documentation pages on the wiki ](https://docs.openspaceproject.com/en/latest/manual/navigation/joystick-customization.html), so maybe not worth the effort?